cnxarchive/sql/search/author.part.sql

```diff
@@ -9,15 +9,15 @@ SELECT
   %({0})s::text||'-::-author' as key
 FROM
   latest_modules AS m,
-  users AS u 
+  users AS u
 WHERE
   u.username = ANY (m.authors)
   AND
-  (u.first_name ~* req(%({0})s::text)
+  (u.first_name ~* regexp_replace(req(%({0})s::text), ' ', '.+', 'g')
```

Comment:
I tested, your way is faster and more correct. Though I stand by only needing it on fullname.